week-1/exercises/initials.js

```diff
@@ -1,6 +1,7 @@
 let firstName = "Creola";
 let middleName = "Katherine";
 let lastName = "Johnson";
-
+const initials = firstName.charAt(0)+ middleName.charAt(0) + lastName.charAt(0)
```

Comment:
Be careful of the requirement "in upper case". Even though the initials are all uppercase in this example, that might not be true for all inputs.